src/collection/components/modals/DeleteCollectionModal.tsx

```diff
@@ -30,6 +30,12 @@ const DeleteCollectionModal: FunctionComponent<DeleteCollectionModalProps> = ({
 
 	const renderDeleteMessageParagraph = () => {
 		if (isSharedWithOthers) {
+			if (contributorCount === 1) {
+				return tHtml(
+					'Ben je zeker dat je jezelf van deze collectie wil wissen? Deze opdracht is met 1 andere persoon gedeeld. Deze verliezen dan toegang.'
```

Comment:
Isn't the owner also a contributor? So if it is shared with 1 person, the count would be 2

Reply:
no, the owner is separate under the owner_profile_id column (owner relationship)
only contributors wirth type CONTRIBUTER and VIEWER are in the contributors table